test/alignment-D3.js

```diff
@@ -116,287 +97,190 @@ describe('D3 Alignment module tests', function() {
     <g class="tick" transform="translate(0,380.5)">
       <text>37:00</text>
     </g>
+    <g class="tick" transform="translate(0,410.5)">
+      <text>36:45</text>
+    </g>
   </g>
-
   <g id="x-axis">
-    <g class="tick" transform="translate(36.52173913043478,0)">
+    <g class="tick" transform="translate(110.91666666666666,0)">
       <text>1994</text>
     </g>
-    <g class="tick" opacity="1" transform="translate(109.56521739130434,0)">
+    <g class="tick" transform="translate(171.75,0)">
       <text>1996</text>
     </g>
-    <g class="tick" opacity="1" transform="translate(182.6086956521739,0)">
+    <g class="tick" transform="translate(232.66666666666666,0)">
       <text>1998</text>
     </g>
   </g>
-
-  <circle class="dot" r="6" cx="36.52173913043478" cy="300"
-    data-xvalue="1994" data-yvalue="Mon Jan 01 1900 00:37:40 GMT-0200 (BRST)">
+  <circle class="dot" r="3" cx="140.83333333333331" cy="400" data-xvalue="1995" 
+    data-yvalue="Mon Jan 01 1900 00:36:50 GMT-0700 (MST)">
   </circle>
-  <circle class="dot" r="6" cx="73.04347826086956" cy="360" data-xvalue="1995"
-    data-yvalue="Mon Jan 01 1900 00:37:01 GMT-0200 (BRST)">
+  <circle class="dot" r="3" cx="201.75" cy="390" data-xvalue="1997" 
+    data-yvalue="Mon Jan 01 1900 00:36:55 GMT-0700 (MST)">
   </circle>
-  <circle class="dot" r="6" cx="146.08695652173913" cy="400"
-    data-xvalue="1997" data-yvalue="Mon Jan 01 1900 00:36:55 GMT-0200 (BRST)">
+  <circle class="dot" r="3" cx="110.41666666666666" cy="350" data-xvalue="1994" 
+    data-yvalue="Mon Jan 01 1900 00:37:15 GMT-0700 (MST)">
   </circle>
-
+  
   <circle class="dot-misaligned" r="6" cx="0"
     cy="355" data-xvalue="1997"
     data-yvalue="Mon Jan 01 1900 00:37:25 GMT-0200 (BRST)">
   </circle>
   `);
 
   const barChartDom = new JSDOM(`
-    <g id="y-axis">
+    <g fill="none" id="y-axis" transform="translate(60, 0)">
       <path class="domain" stroke="#000" d="M-6,400.5H0.5V5.882873781463295H-6">
       </path>
+      <g class="tick" opacity="1" transform="translate(0,361.597610256467)">
+        <line stroke="#000" x2="-6"></line>
```

Comment:
I'm pretty sure we don't need the `line` elements for the tests to work? In general we should try to keep test data to only the minimum required because it makes it easier to understand what data the functions require.

I'm going to stop commenting here about any other stuff that's not needed to pass the tests, but it would be good if you can strip all of the test data to the bare minimum needed by the tests.